frontend/src/components/Search/SimpleSearch/SimpleSearch.logic.ts

```diff
@@ -0,0 +1,49 @@
+import type { QueryParams } from "~/stores/searchParams";
+import { DocumentKind } from "~/types";
+
+export function getUrl(category?: string) {
+  if (category?.startsWith(DocumentKind.CaseLaw)) {
```

Comment:
Isn't the category supposed to be exactly equal the value in the DocumentKind.CaseLaw? :thinking: using startsWith might be dangerous for maintainability of the code or for erroneous strings since the category string type is unchecked about (it's only string)

Reply:
I've added a comment here, this can be "R" or things like "R.Urteil". But we should consider moving all these to the enum, you are right.